recipes/liboptv/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
```

Comment:
Using a variable will allow the bot to automatically make update pull requests and ensure things don't get out of sync:
```suggestion
{% set version = "0.2.3" %}

package:
```